src/config/index.ts

```diff
@@ -0,0 +1,21 @@
+export const getEnvironmentValue = (key: string, defaultValue = ""): string => {
+  const value: string = process.env[key] || defaultValue;
+
+  if (!value) {
+    throw new Error(`Please set the environment variable "${key}"`);
+  }
+
+  return value;
+};
+
+// APP CONFIGs
+export const APPLICATION_NAME = "overseas-entities-web";
```

Comment:
I can't see where this global constant is used. Is it required (yet)?

Reply:
It is not used yet, but it will be soon essential when we integrate the` @companieshouse/structured-logging-node`

Reply:
Ok, not an issue, but suggest we only add code at the point it's needed and (ideally) each line of code should somehow track back to a requirement in the story or the task the PR is related to. Sorry, I get a bit obsessed about this!

Reply:
Just seen you've removed this. Ignore my comment and thanks!